### PR TITLE
fix: signingscript notarization - fix credential handling

### PR DESCRIPTION
**Pull request by hneiva**

### CODE REVIEW

**Review by gabrielbusta — APPROVED** [review]

File: signingscript/src/signingscript/utils.py [hunk 1]
```diff
@@ -92,11 +92,15 @@ def _load_scoped_configs(filename, cls, name):
     with open(filename) as f:
         raw_cfg = json.load(f)
 
-    cfg = {}
+    scope_configs = {}
+    # TODO: We should refactor how Autograph data is loaded to be key-pair instead of list
```
Comment (gabrielbusta) [inline signingscript/src/signingscript/utils.py:96]: Yea

File: signingscript/src/signingscript/utils.py [hunk 2]
```diff
@@ -92,11 +92,15 @@ def _load_scoped_configs(filename, cls, name):
...
     for scope, config in raw_cfg.items():
-        cfg[scope] = [cls(*s) for s in config]
+        if cls == Autograph:
+            scope_configs[scope] = [cls(*s) for s in config]
+        elif cls == AppleNotarization:
```
Comment (gbrownmozilla) [inline signingscript/src/signingscript/utils.py:100]: What if it's something other than Autograph or AppleNotarization? (Could we make this an if/else?)

File: signingscript/src/signingscript/utils.py (hunk 1 above)
Comment (gabrielbusta) [reply signingscript/src/signingscript/utils.py:96]: If we refactored that you would not need the if else on the `cls` instantiation?

File: signingscript/src/signingscript/utils.py (hunk 2 above)
Comment (hneiva) [reply signingscript/src/signingscript/utils.py:100]: Good point, I'll raise an exception for now

**Pull request merged**